stories/CaseBlockStyled.stories.ts

```diff
@@ -32,44 +32,32 @@ const CaseBlockTemplate = `
 const BaseOptions = {
   components: {CaseBlock, FlowBuilderSidebarEditorContainer},
   template: CaseBlockTemplate,
+  store: new Vuex.Store<IRootState>(store),
 }
 
 // default case block state
 @Component<any>({
   ...BaseOptions,
-  store: new Vuex.Store<IRootState>(store),
-  async mounted() {
-    // @ts-ignore
-    await baseMounted.bind(this)(BLOCK_TYPE, caseBlockStore)
-  },
 })
-class DefaultClass extends BaseMountedVueClass {}
+class DefaultClass extends BaseMountedVueClass {
+  async mounted() {
+    await this.baseMounted(BLOCK_TYPE, caseBlockStore)
+  }
+}
 export const Default = () => (DefaultClass)
 
 //ExistingDataBlock
 @Component<any>({
   ...BaseOptions,
-  store: new Vuex.Store<IRootState>(store),
   async mounted() {
-    // @ts-ignore
-    const { block } = await baseMounted.bind(this)(BLOCK_TYPE, caseBlockStore)
+    const { block } = await this.baseMounted(BLOCK_TYPE, caseBlockStore)
     const blockId = block.uuid
 
-    // @ts-ignore - TS2339: Property 'block_setName' does not exist on type
-    this.block_setName({blockId: blockId, value: "A Name"})
-    // @ts-ignore - TS2339: Property 'block_setLabel' does not exist on type
-    this.block_setLabel({blockId: blockId, value: "A Label"})
-    // @ts-ignore - TS2339: Property 'block_setSemanticLabel' does not exist on type
-    this.block_setSemanticLabel({blockId: blockId, value: "A Semantic Label"})
-    // @ts-ignore - TS2339: Property 'editCaseBlockExit' does not exist on type
+    this.setDescription(blockId)
```

Comment:
As the vue-class extends is working now, I moved repeated code into `story-utils/storeSetup.ts`'s classes